### DISCUSSION TITLE
Excluding / Removing overlapping segment for speaker-diarization

### DISCUSSION THREAD

**Discussion by hkpmatt** [pyannote/pyannote-audio#1779] [General]
I am using the speaker-diarization pipeline and I have met some problem where there should be no overlapping speech in the audio but the segmentation model seems to falsely detected it. The only parameter for the "pyannote/segmentation-3.0" only have 1 hyperparameter that is the min_duration_off which will not settle the problem. Is there any parameter to remove or exclude the overlapping segment? 
![output](https://github.com/user-attachments/assets/cf2d6508-3ce9-441c-8745-089ac1f12b39)

**Comment by hbredin** [comment]
Closing as duplicate of #1780
